src/metabase/models/query.clj

```diff
@@ -38,21 +38,23 @@
 (defn- update-rolling-average-execution-time!
   "Update the rolling average execution time for query with `query-hash`. Returns `true` if a record was updated,
    or `false` if no matching records were found."
-  ^Boolean [query, ^bytes query-hash, ^Integer execution-time-ms]
-  (let [avg-execution-time (hx/cast (int-casting-type) (hx/round (hx/+ (hx/* 0.9 :average_execution_time)
-                                                                       (*    0.1 execution-time-ms))
-                                                                 0))]
+  ^Boolean [query ^bytes query-hash ^Integer execution-time-ms]
+  (let [avg-execution-time (h2x/cast (int-casting-type) (h2x/round (h2x/+ (h2x/* [:inline 0.9] :average_execution_time)
+                                                                          [:inline (* 0.1 execution-time-ms)])
+                                                                   [:inline 0]))]
 
     (or
      ;; if it DOES NOT have a query (yet) set that. In 0.31.0 we added the query.query column, and it gets set for all
      ;; new entries, so at some point in the future we can take this out, and save a DB call.
-     (db/update-where! Query {:query_hash query-hash, :query nil}
-       :query                 (json/generate-string query)
-       :average_execution_time avg-execution-time)
+     (db/update-where! Query
+                       {:query_hash query-hash, :query nil}
+                       :query                 (json/generate-string query)
+                       :average_execution_time avg-execution-time)
```

Comment:
In case it wasn't clear yet, I changed the `:style/indent` specs in Toucan 2 to actually make sense and follow rules more consistently. So there are a lot of indentation changes like this